cmd/influxd/inspect/verify_series.go

```diff
@@ -0,0 +1,91 @@
+package inspect
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/influxdata/influxdb/logger"
+	"github.com/influxdata/influxdb/tsdb"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap/zapcore"
+)
+
+// NewVerifySeriesFileCommand returns a new instance of verifySeriesCommand
+// for execution of "influx_inspect verify-seriesfile".
+func NewVerifySeriesFileCommand() *cobra.Command {
+	verifySeriesCommand := &cobra.Command{
+		Use:   "verify-seriesfile",
+		Short: "Verifies the integrity of Series files",
+		Long: `Verifies the integrity of Series files.
+		Usage: influx_inspect verify-seriesfile [flags]
+			-dir <path>
```

Comment:
Since we only support a single series file per data directly, we don't need a `--dir` option